src/index.css

```diff
@@ -1,27 +1,45 @@
-body {
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: rgb(215, 215, 224);
+}
+
+.calculator__container {
+  display: grid;
+  grid-template-columns: repeat(4, 6rem);
+  grid-template-rows: minmax(5rem, auto) repeat(5, 5rem);
+  margin-top: 4rem;
+  justify-content: center;
+}
+
+.calculator__container button {
+  outline: none;
+  cursor: pointer;
+  border: 1px solid rgb(209, 209, 202);
+  font-size: 1rem;
+}
+
+.calculator__container button:hover,
+.calculator__container button:focus {
+  background-color: gray;
+}
+
+.span__two {
+  grid-column: span 2;
+}
+
+.calculator__output {
+  grid-column: 1/-1;
+  background-color: gray;
+  color: #fff;
+  padding: 1rem;
+  font-size: 3rem;
```

Comment:
- [ ] For a better user interface and to be similar to the expected UI, please align the text to the right `text-align: end;`👇🏽 

| Expected | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212223406-e1f8c5a0-5436-423b-8eb9-680cb73d400e.png) | ![image](https://user-images.githubusercontent.com/58642949/212223472-202ef3aa-47b9-4e2c-b9ff-5e08fffaacba.png) |